.cloudbuild.sh

```diff
@@ -1,11 +1,8 @@
 #! /bin/bash
 
-. release-tools/prow.sh
-
-# Extract tag-n-hash value from GIT_TAG (form vYYYYMMDD-tag-n-hash) for REV value.
-REV=v$(echo "$GIT_TAG" | cut -f3- -d 'v')
+: ${CSI_PROW_BUILD_PLATFORMS:="windows amd64 .exe nanoserver:1809; windows amd64 .exe nanoserver:ltsc2022"}
+: ${REGISTRY_NAME:="gcr.io/k8s-staging-sig-storage"}
```

Comment:
This might make the presubmit job pass but I think I might be forgetting about the post submit image push to `gcr.io/k8s-staging-sig-storage`, I'll check that too

Reply:
```
: ${REGISTRY_NAME:="gcr.io/k8s-staging-sig-storage"}
```

shouldn't be set in this file, this is the file that runs on postsubmit